GameOfLife/board.cpp

```diff
@@ -1,11 +1,33 @@
 #include "board.h"
 
-Board::Board()
+int Board::getNumberOfCollumn() const
 {
+    return numberOfCollumn;
+}
 
+void Board::setNumberOfCollumn(int value)
+{
+    numberOfCollumn = value;
+}
+
+int Board::getNumberOfRow() const
+{
+    return numberOfRow;
+}
+
+void Board::setNumberOfRow(int value)
+{
+    numberOfRow = value;
+}
```

Comment:
The setter functions are incorrect. They are public, and can be used on the board object from any code outside.
The function just change the value of parameters without actually setting the size of the board.

Something like this might be written
Board test(2,2);
test.setNumberOfRows(5);
test.getTheBoard()[4][1];